examples/apps/collaborative-textarea/jest.config.js

```diff
@@ -20,4 +20,7 @@ module.exports = {
 	transform: {
 		"^.+\\.ts?$": "ts-jest",
 	},
+	// While we still have transitive dependencies on 'uuid<9.0.0', force the CJS entry point:
+	// See: https://stackoverflow.com/questions/73203367/jest-syntaxerror-unexpected-token-export-with-uuid-library
+	moduleNameMapper: { "^uuid$": "uuid" },
```

Comment:
Unfortunately, for a handful of projects I needed to force the CJS entry point.

Reply:
if we are going to keep this in our jest config, it might make sense to do it in the shared config in test-tools instead of in each package impacted.